int32 row
int32 column
bool isBlack
---
bool success
